docs/network-flow-visibility.md

```diff
@@ -831,8 +885,9 @@ Currently we only support the visualization of NetworkPolicies with `Allow` acti
 
 If you would like to make any changes to any of the pre-built dashboards, or build
 a new dashboard, please follow this [doc](https://grafana.com/docs/grafana/latest/dashboards/export-import/)
-for dashboard export and import. To generate a deployment manifest with the changes,
-please follow the following steps:
+for dashboard JSON files export and manual import.
+
+To generate a deployment manifest with the changes, please follow the following steps:
```

Comment:
maybe emphasize that there are 2 ways of adding dashboards:
* in the UI or by importing a JSON at runtime (I don't know how these are persisted to Grafana, they may be lost in case of a Pod restart?)
* by generating a new manifest and (re)deploying, as described below

Reply:
yes, the changes will be lost after a Pod restart. They won't be automatically saved to the source.
I have added the 2 ways, could you help check if the wordings make sense to you? Thanks